CommunityFridgeMapApi/functions/email_service/v1/app.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import json
+
+from send_email import sendEmail
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+recipient = "info@collectivefocus.site"
+
+
+class ContactUs:
+    @staticmethod
+    def lambda_handler(event: dict) -> dict:
+        body = event.get("body", {})
+        if not body:
+            return {"statusCode": 400}
+        body = json.loads(body)
+        sender = body.get("name", None)
+        senderAddress = body.get("email", None)
+        subject = body.get("subject", "no subject")
+        message = body.get("message", None)
+
+        if not sender or not senderAddress or not message:
+            raise ValueError('missing required field to send email')
```

Comment:
Could we change the response to be consistent with the other APIs?

{
  "statusCode": 400,
  "headers": {
      "Content-Type": "application/json",
      "Access-Control-Allow-Origin": "*",
  },
  "body": {"message": MESSAGE},
}